src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json

```diff
@@ -861,27 +920,23 @@
                     "ReturnId": "6280",
                     "Conditions": [
                         {
-                            "QuestionTag": "ApplyProviderRoute",
-                            "MustEqual": "2"
+                            "QuestionTag": "PATTypeOfTrainingMainEmployer",
+                            "MustEqual": "Apprenticeship frameworks"
                         },
                         {
-                            "QuestionTag": "PATTypeOfTrainingMainEmployer",
-                            "Contains": "Apprenticeship frameworks"
+                            "QuestionTag": "PATPlanningToTransitionToStandards",
+                            "MustEqual": "Yes"
                         }
                     ]
                 },
                 {
                     "Action": "NextPage",
-                    "ReturnId": "6280",
+                    "ReturnId": "6253",
                     "Conditions": [
-                        {
-                            "QuestionTag": "ApplyProviderRoute",
-                            "MustEqual": "1"
-                        },
-                        {
-                            "QuestionTag": "PATTypeOfTrainingMainEmployer",
-                            "Contains": "Apprenticeship frameworks"
-                        }
+						            {
+							              "QuestionTag": "PATTypeOfTrainingMainEmployer",
+							              "Contains": "Apprenticeship frameworks"
+						            }
                     ]
```

Comment:
just to confirm that these change in Next conditions are fine as I've not looked at the ticket but it's no longer checking provider route